dashboards-observability/public/components/visualizations/charts/data_table/data_table.tsx

```diff
@@ -46,6 +33,30 @@ export const DataTable = ({ visualizations, layout, config }: any) => {
     metadata: { fields = [] },
   } = visualizations.data.rawVizData;
 
+  const { dataConfig = {} } = visualizations?.data?.userConfigs;
```

Comment:
Can we follow bar's viz data destruction code here to destruct data config so we don't have to make these multiple long lines of assignments?